stl/inc/stacktrace

```diff
@@ -0,0 +1,347 @@
+// stacktrace standard header
+
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#pragma once
+#ifndef _STACKSTRACE_
+#define _STACKSTRACE_
+#include <yvals_core.h>
+#if _STL_COMPILER_PREPROCESSOR
+
+#if !_HAS_CXX23
+#pragma message("The contents of <stacktrace> are available only with C++23 or later.")
+#else // ^^^ !_HAS_CXX23 / _HAS_CXX23 vvv
+
+#include <cstdint>
+#include <string>
+#include <type_traits>
+#include <vector>
+
+#pragma pack(push, _CRT_PACKING)
+#pragma warning(push, _STL_WARNING_LEVEL)
+#pragma warning(disable : _STL_DISABLED_WARNINGS)
+_STL_DISABLE_CLANG_WARNINGS
+#pragma push_macro("new")
+#undef new
+
+using _Stacktrace_string_fill_callback = size_t (*)(char*, size_t, void* _Context);
+
+using _Stacktrace_string_fill = size_t (*)(size_t, void* _String, void* _Context, _Stacktrace_string_fill_callback);

```

Comment:
Suggestion: Right now, these typedefs provide names for some, but not all, parameters. It would seem to be more consistent to provide names for all parameters.